Logger.lua

```diff
@@ -92,11 +83,15 @@ function Logger:new(name, dir, console)
   if not dir or  #dir == 0 then o.Path = "./"
   elseif dir and DirExists(dir) then o.Path = DirNormalize(dir)
   elseif dir and not DirExists(dir) then
-    error("Path '" .. dir .. "' doesn't exists or you can't have permissions to use it.")
+    error("Path '" .. dir .. "' doesn't exists or you don't have permissions to use it.")
   else -- idk...
-    error("Something's wrong with '"..dir.."'... (argument #2 in 'new()')")
+    error("Unknown error while checking (and/or loading) '"..dir.."'... (argument #2 in 'new()')")
```

Comment:
i wanted it this way because it is more specific about the problem of not knowing the problem instead of saying "dude there is a problem"

me gusto asi porque era mas especifico diciendo que no sabia cual era el problema en vez de decir que hay un problema y no especificar que

Reply:
Correcto, así tiene más sentido